src/System.Drawing.Primitives/src/System/Drawing/PointF.cs

```diff
@@ -38,115 +34,88 @@ public PointF(float x, float y)
 
         /// <summary>
-        ///    <para>
-        ///       Compares two <see cref='System.Drawing.PointF'/> objects. The result specifies whether the values
-        ///       of the <see cref='System.Drawing.PointF.X'/> or <see cref='System.Drawing.PointF.Y'/> properties of the two
-        ///    <see cref='System.Drawing.PointF'/> 
-        ///    objects are unequal.
-        /// </para>
+        /// Compares two <see cref='System.Drawing.PointF'/> objects. The result specifies whether the values of the
+        /// <see cref='System.Drawing.PointF.X'/> or <see cref='System.Drawing.PointF.Y'/> properties of the two
+        /// <see cref='System.Drawing.PointF'/> objects are unequal.
         /// </summary>
         public static bool operator !=(PointF left, PointF right) => !(left == right);
 
         /// <summary>
-        ///    <para>
-        ///       Translates a <see cref='System.Drawing.PointF'/> by a given <see cref='System.Drawing.Size'/> .
-        ///    </para>
+        /// Translates a <see cref='System.Drawing.PointF'/> by a given <see cref='System.Drawing.Size'/> .
         /// </summary>
         public static PointF Add(PointF pt, Size sz) => new PointF(pt.X + sz.Width, pt.Y + sz.Height);
 
         /// <summary>
-        ///    <para>
-        ///       Translates a <see cref='System.Drawing.PointF'/> by the negative of a given <see cref='System.Drawing.Size'/> .
-        ///    </para>
+        /// Translates a <see cref='System.Drawing.PointF'/> by the negative of a given <see cref='System.Drawing.Size'/> .
         /// </summary>
         public static PointF Subtract(PointF pt, Size sz) => new PointF(pt.X - sz.Width, pt.Y - sz.Height);
 
         /// <summary>
-        ///    <para>
-        ///       Translates a <see cref='System.Drawing.PointF'/> by a given <see cref='System.Drawing.SizeF'/> .
-        ///    </para>
+        /// Translates a <see cref='System.Drawing.PointF'/> by a given <see cref='System.Drawing.SizeF'/> .
         /// </summary>
         public static PointF Add(PointF pt, SizeF sz) => new PointF(pt.X + sz.Width, pt.Y + sz.Height);
 
         /// <summary>
-        ///    <para>
-        ///       Translates a <see cref='System.Drawing.PointF'/> by the negative of a given <see cref='System.Drawing.SizeF'/> .
-        ///    </para>
+        /// Translates a <see cref='System.Drawing.PointF'/> by the negative of a given <see cref='System.Drawing.SizeF'/> .
         /// </summary>
         public static PointF Subtract(PointF pt, SizeF sz) => new PointF(pt.X - sz.Width, pt.Y - sz.Height);
 
         public override bool Equals(object obj) => obj is PointF && Equals((PointF)obj);
 
         public bool Equals(PointF other) => this == other;
 
-        public override int GetHashCode() => HashHelpers.Combine(X.GetHashCode(), Y.GetHashCode());
+        public override int GetHashCode() => HashCode.Combine(X, Y);
```

Comment:
These changes from HashHelpers.Combine to HashCode.Combine are adding randomization into hashes and making hash codes more expensive.  Are the benefits there actually worth the costs?  What is the benefit of randomization in a case like this?

Reply:
If I’m to be honest, I wasn’t aware that there was a difference. I can try testing the performance - I was under the impression that they’d be equivalent as we tout HashCode.Combine as a high perf api? It’s weird that we use an internal HashHelpers API and then expose a HashCode API that we don’t use internally

Reply:
https://github.com/dotnet/coreclr/issues/26582